docs/lambda_versioning.md

```diff
@@ -40,6 +40,8 @@ SomeLambda:
 
 That key, due to AWS constraints, must be letters (```[a-zA-Z]```) only.
 
+Note that if the lambda is configured to run in a VPC and the VPC settings change, you will need to manually update the `uniqueIdentifier`.
```

Comment:
We talked about this this morning, we should probably change the wording to suggest that it will need to actually re-deploy.....or remove pending the new ticket being written against the behavior?

Reply:
Well... hmmmm.... I removed it, but then realized the line above actually works. When I deployed with the new VPC things broke, but manually updating the unique Id worked for master because the master stack was already existing with the old VPC.

The problem is it didn't work for master AND our stacks. Our stacks had been initially deployed with the new VPC and updating the unique id did not work for those since nothing had changed.

So the problem isn't really that we can't update when we change the VPC, it's that we didn't have a solution that worked when having multiple stacks in different states.

Reply:
I think there is a bug here for sure, but maybe not as bad as I initially thought.

Reply:
https://bugs.earthdata.nasa.gov/browse/CUMULUS-1153

Reply:
I think in either case, we don't want just the uniqueIdentifier to change as that will result in the build not working as the payload for the lambda didn't change and it will have a version collision.

Reply:
....but apparently in the case of S3 objects does not?     Interesting.  I'll comment on 1153.   This does work for now.